repos/system_upgrade/el7toel8/models/sctpconfigmodel.py

```diff
@@ -0,0 +1,6 @@
+from leapp.models import Model, fields
+from leapp.topics import SCTPConfigTopic
+
+class SCTPConfig(Model):
+    topic = SCTPConfigTopic
+    wanted = fields.Boolean(False)
```

Comment:
```suggestion
    wanted = fields.Boolean(default=False)
```
Better to specify the parameter name to guarantee it's the right field which is assigned in case of some refactoring or parameter placement changes.